src/content/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "NerdGraph tutorial: Browser agent monitoring examples"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+  - Browser
+  - Mobile
+metaDescription: Examples of using New Relic NerdGraph API to adjust a browser agent
+---
+
+You can create browser applications in the NerdGraph API instead of using the UI. The advantage to this is that you can programmatically create and retrieve the JavaScript snippet to copy and paste into your browser application.
+
+## Create a new browser application [#create-browser]
+
+The following example mutation will create a new browser application with default settings.
+
+```graphql
+mutation CreateExampleBrowserApplication {
+  agentApplicationCreateBrowser(accountId: YOUR_ACCOUNT_ID, name: "Example Browser Application", settings: {cookiesEnabled: true, distributedTracingEnabled: true, loaderType: SPA}) {
+    guid
+    name
+    settings {
+      cookiesEnabled
+      distributedTracingEnabled
+      loaderScript
+      loaderType
+    }
+  }
+}
+```
+
+## Retrieve the JavaScript snippet [#get-browser-snippet]
+
+You can retrieve the JavaScript snippet to [copy/paste into your application](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app). Note that the returned snippet is a JSON encoded string that will need to be parsed before it can be copy/pasted.
```

Comment:
```suggestion
You can retrieve the JavaScript snippet to [copy/paste into your application](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app). Note that the returned snippet is a JSON encoded string that will need to be parsed before it can be copy/pasted.
```